README.md

```diff
@@ -1,2 +1,266 @@
-# lb4-log-extension-example
+# loopback4-example-log-extension
 An example repo showing how to write a complex log extension
+
+## Overview
+
+This repository shows you how to use [loopback4-extension-starter]() to write a complex logging extension that requires a [Component](), [Decorator](), and a [Mixin]().
+
+To use the extension, the user would load the component to get access to a `LogFn` that can be used in a sequence to log information. A Mixin allows the user to set the application wide logLevel. Only Controller methods configured at or above the logLevel will be logged. 
```

Comment:
"To use the extension, load the component ..."

"A Mixin allows you to set ..."